modules/client/src/connext.ts

```diff
@@ -193,10 +196,12 @@ export async function connect(opts: ClientOptions): Promise<ConnextInternal> {
  */
 export abstract class ConnextChannel {
   public opts: InternalClientOptions;
+  public config: GetConfigResponse;
   private internal: ConnextInternal;
 
   public constructor(opts: InternalClientOptions) {
     this.opts = opts;
+    this.config = opts.config;
```

Comment:
Fetch the node's config once on connection and then expose the result to use synchronously later w/out needing to await a msg from the node every time we want to access config. I also removed the async getConfig method in favor of the config property because this config stuff (urls & contract addresses mostly) doesn't change